From 69b2679eac4303df0790363bb9bbf7a7b556ebdd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jani=20V=C3=A4limaa?= <wally@mageia.org>
Date: Sun, 17 Dec 2017 14:47:24 +0200
Subject: [PATCH] Fix web-extensions location

---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1e4213f..5e2abb8 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -745,7 +745,7 @@ OPTIONS
 
 add_library(geary-web-process MODULE ${WEB_PROCESS_VALA_C})
 target_link_libraries(geary-web-process PRIVATE ${DEPS_LIBRARIES} geary-engine)
-install(TARGETS geary-web-process LIBRARY DESTINATION lib/geary/web-extensions)
+install(TARGETS geary-web-process LIBRARY DESTINATION lib${LIB_SUFFIX}/geary/web-extensions)
 
 # Console app
 #################################################
-- 
2.15.1

